hmda/src/main/scala/hmda/validation/rules/lar/validity/2022/V665_2.scala

```diff
@@ -0,0 +1,17 @@
+package hmda.validation.rules.lar.validity._2022
+
+import hmda.model.filing.lar.LoanApplicationRegister
+import hmda.model.filing.lar.enums._
+import hmda.validation.dsl.PredicateCommon._
+import hmda.validation.dsl.PredicateSyntax._
+import hmda.validation.dsl.ValidationResult
+import hmda.validation.rules.EditCheck
+
+object V665_2 extends EditCheck[LoanApplicationRegister] {
+  override def name: String = "V665-2"
+
+  override def parent: String = "V665"
+
+  override def apply(lar: LoanApplicationRegister): ValidationResult =
+    lar.coApplicant.creditScoreType not equalTo(new InvalidCreditScoreCode)
```

Comment:
I think it would be best if we checked against a full list of available options incase new options are added in the future.

For example, if an option 12 was added next year this edit would allow that option for 2022 which would be incorrect.

Reply:
@kgudel  Pushed up a change for this